composer.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "prestashop/ps_healthcheck",
+  "description": "PrestaShop - Health check",
+  "homepage": "https://github.com/PrestaShop/ps_healthcheck",
+  "license": "AFL-3.0",
+  "authors": [
+    {
+      "name": "PrestaShop SA",
+      "email": "contact@prestashop.com"
+    }
+  ],
+  "require": {
+    "php": ">=5.6.0",
```

Comment:
Can be >= 7.1.3 as the Core, not sure it will working under 1.7.7.0

Reply:
done